src/core/Microsoft.PowerToys.Settings.UI.Lib/ImageResizerSettings.cs

```diff
@@ -2,33 +2,26 @@
 // The Microsoft Corporation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
 namespace Microsoft.PowerToys.Settings.UI.Lib
 {
-    public class ImageResizerSettings
+    public class ImageResizerSettings : BasePTModuleSettings
     {
-        [JsonPropertyName("version")]
-        public string Version { get; set; }
-
-        [JsonPropertyName("name")]
-        public string Name { get; set; }
+        public const string ModuleName = "Image Resizer";
```

Comment:
BasePTModuleSettings already has the property Name.

Reply:
That is why it was removed